### ISSUE TITLE
Enable GitHub Discussions

### ISSUE THREAD

**Issue by webknjaz** [pypa/setuptools#2557]
@jaraco there are a lot of Q&A style questions in the list of issues. I think they should go to their own tab called `Discussions` that also allows marking the correct answers. It's also possible to convert existing issues into discussions so the bugs and feature requests list would be separate from the StackOverflow-like pages.

Demo: https://github.com/pypa/gh-action-pypi-publish/discussions

To enable this, go to https://github.com/pypa/setuptools/settings#features and toggle the `Discussions` checkbox under "Features".

**Comment by jaraco** [comment]
I'm a little worried adding yet another forum is not what this project needs, but I'm happy to enable it experimentally. I can't promise I'll be able to participate much there. It's enabled now.

**Comment by webknjaz** [comment]
I think the point is that the community can interact there w/o flooding the bug tracker.

**Comment by hroncok** [comment]
Is somebody following the topis there? It does not seem to work.

**Comment by webknjaz** [comment]
@hroncok It works (https://github.com/pypa/setuptools/discussions) but the issue forms are broken (https://github.com/pypa/setuptools/pull/2649).

**Comment by hroncok** [comment]
> It works (https://github.com/pypa/setuptools/discussions) ...

Yes, but there is no discussion happening there. Only questions and no replies.

**Comment by webknjaz** [comment]
Ah, I think it's because the maintainers don't really have much time to do this 24x7. I think even I (not a maintainer in this repo) look into the PRs more. And I look into the issues occasionally mostly when I have the repo open for some other reason. I think @jaraco also looks into the PRs more but also he has hundreds of other projects to watch after.
It's not the fault of discussions being a separate space, in my opinion. But then, the bug tracker has a different purpose and thus is often more important to look into...
I've just noticed that I'm `Watching` only the releases in this repo so I went ahead and added `Discussions` under `Custom` so I'll get those in my inbox too.

**Comment by webknjaz** [comment]
Also, it's worth mentioning that `Discussions` is supposed to be a StackOverflow-like place for the community to interact rather than a place to demand the maintainer support.

**Comment by hroncok** [comment]
So if there is a discussion I want to have *with the maintainers*, should I use issues instead?

**Comment by hroncok** [comment]
Note that the *New issue* button does not seem to list ahe regular "issue", but the direct link https://github.com/pypa/setuptools/issues/new/ works.

**Comment by webknjaz** [comment]
> Note that the _New issue_ button does not seem to list the regular "issue", but the direct link [pypa/setuptools/issues/new](https://github.com/pypa/setuptools/issues/new/) works.

This is because of a bug in the schema files, once https://github.com/pypa/setuptools/pull/2649 is merged, this will be fixed.

**Comment by webknjaz** [comment]
> So if there is a discussion I want to have _with the maintainers_, should I use issues instead?

I suggest creating a discussion and tagging @jaraco (I just don't know if he has those notifications by default). Also, now that I'm subscribed to the `Discussions` notifications, I should be able to notice such things and maybe tag somebody in too.

**Comment by jaraco** [comment]
Indeed, I have notifications disabled by default. I make a point of following up on every e-mail I get, but I can only periodically get to the backlog of issues, so I don't subscribe to each issue or discussion. I haven't yet engaged with the discussion groups and I welcome content there, although perhaps discuss is a better forum. Sorry for the absence.

**Comment by webknjaz** [comment]
@jaraco may I suggest you to subscribe just to discussions as an experiment? They are extremely low traffic (just 3 discussions started) so they won't spam your inbox too much but the benefit would be that you'll notice when there's some activity (w/o an expectation to engage right away).

Here's how this looks for me:
![setuptools-notifications](https://user-images.githubusercontent.com/578543/116634018-4b936100-a95b-11eb-8764-ccdd83d890be.png)
